README.md

```diff
@@ -42,11 +42,28 @@ helm install "${SKYWALKING_RELEASE_NAME}" \
   oci://registry-1.docker.io/apache/skywalking-helm \
   --version "${SKYWALKING_RELEASE_VERSION}" \
   -n "${SKYWALKING_RELEASE_NAMESPACE}" \
-  --set oap.image.tag=10.0.0 \
+  --set oap.image.tag=10.0.1 \
   --set oap.storageType=elasticsearch \
-  --set ui.image.tag=10.0.0
+  --set ui.image.tag=10.0.1
 ```
 
+To use banyandb as storage solution, you can try
+
+```shell
+helm install "${SKYWALKING_RELEASE_NAME}" \
+  oci://registry-1.docker.io/apache/skywalking-helm \
+  --version "${SKYWALKING_RELEASE_VERSION}" \
+  -n "${SKYWALKING_RELEASE_NAMESPACE}" \
+  --set oap.image.tag=10.0.1 \
+  --set oap.storageType=banyandb \
+  --set ui.image.tag=10.0.1 \
+  --set elasticsearch.enabled=false \
+  --set banyandb.enabled=true \
+  --set banyandb.image.tag=0.6.1
+```
+
+The parameter `banyandb.image.tag` can be used to specify the version of banyandb.
```

Comment:
Could you add docs about how to use all available BanyanDB parameters in https://github.com/apache/skywalking-banyandb-helm?tab=readme-ov-file#configuration ?